101-iothub-device-provisioning/azuredeploy.json

```diff
@@ -58,7 +63,7 @@
                 "capacity": "[parameters('skuUnits')]"
             },
             "properties": {
-              "minTlsVersion": "[if(parameters('requireTLS'), variables('minTlsVersion'), json('null'))]"
+              "minTlsVersion": "[variables('minTlsVersion')]"
```

Comment:
Now we have hard-coded TLS support where only 1.2 will possibly work and a large part of our customers may have legacy devices that can't connect because of no device TLS support.  We should default to TLS off

Reply:
Ok, let's just see if we can create a README.md that users can understand.  Once that's clear we can figure out the template.  Here's my 8th attempt at trying to understand this:

-This template always deploys DPS
-DPS only supports TLS 1.2
-DPS does not require TLS except in usgovvirginia (at least for now)

IoTHub support for TLS is somewhat orthogonal since this template uses DPS and that has stricter requirements.